guides/v2.2/config-guide/cli/config-cli-subcommands-static-view.md

```diff
@@ -1,18 +1,8 @@
 ---
 group: config-guide
-subgroup: 04_CLI
 title: Deploy static view files
-menu_title: Deploy static view files
-menu_node:
-menu_order: 302
-level3_menu_node: level3child
-level3_subgroup: static_deploy
 version: 2.2
 github_link: config-guide/cli/config-cli-subcommands-static-view.md
-functional_areas:

```

Comment:
We need the `functional_areas` tags for Algolia search faceting

Reply:
Gotcha. I didn't see that in the front matter doc I referenced [here](https://wiki.corp.magento.com/display/WRI/Part+1.+Author+in+Markdown+-+Get+Started#Part1.AuthorinMarkdown-GetStarted-Frontmatter). I'll make this change and update the doc!